ls1intum/Artemis
# [Exam] Start and Final Page Component

### Checklist

### Motivation and Context


The exam consists of a starting page (shown before exam has actually started) and a final page (shown after exam has started). 

Resolves #1578 #1582 
### Description

I created a component for displaying the general information and confirmation text produced with the markdown editor. As well, I added a checkbox for confirmation, which if not checked disables the start exam button e.g..

Hint: The title would usually be displayed above the general information text, however it is not possible to do so yet (thus just imagine it were there for now)
### Steps for Testing

_has to be tested locally until exams can be created_

1. Checkout this branch locally
2. Connect to the Artemis database with tablePlus (or whichever tool you prefer)
3. Create a few exams and assign them to a course to play around with
4. By default, the startView will be shown (for testing, you can change this in the code by setting the input boolean of the `exam-participation.component.html` to false in `[startView]=""`) 
5.1. Check that the submit button (Start Exam and Summary) is disabled, when the confirmation box is not checked
5.2. Check that the submit button (Start Exam) turns enabled, when the exam has started and the confirmation box is checked
5.2. Check that the submit button (Start Exam) stays disabled, when the exam has started and the confirmation box is not checked
6. Check whether text entered using the markdown editor is displayed correctly + centered

Hint: "Start Exam" and "Summary Button" to be used and implemented in further development
### Screenshots


After clicking on an exam (e.g. like here) (check out #1628 )....
![image](https://user-images.githubusercontent.com/50466283/84603207-d88d1300-ae8c-11ea-8d76-00c7d1062535.png)

...the following should render (the text depends on the value you gave the exam for startText etc.):
**START view**
![image](https://user-images.githubusercontent.com/50466283/84601429-4bdc5800-ae80-11ea-9dd4-fb5e7392d8c6.png)

This is what it will look like with title
![image](https://user-images.githubusercontent.com/50466283/84600650-3dd80880-ae7b-11ea-8dd7-6b50fd4d5d3b.png)

**FINAL view**
![image](https://user-images.githubusercontent.com/50466283/84601904-e1c5b200-ae83-11ea-8c42-1149bbf08516.png)
